### PR TITLE
Added square example (#41)

### PR DESCRIPTION
**Pull request by biplobsd**
Closed: #41

## Summary
Create contract in the `customersApi` api

## Screen record
[Square-example.webm](https://github.com/triggerdotdev/api-reference/assets/43641536/391a5a24-94d0-4afd-bfcf-fa78b8b9d525)

### CODE REVIEW

File: src/square.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        }),
+    }),
+    run: async (payload, io, ctx) => {
+
+        // Wrap an SDK call in io.runTask so it's resumable and displays in logs
+        await io.runTask(
+            "Square create customer",
+            async () => {
+                // Use the Square SDK to send a message to the seller
+                const customersApi = squareClient.customersApi;
```
Comment (nicktrn) [inline src/square.ts:35]: Don't quite get this extra step, please see the next comment.

File: src/square.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+                // After create a customer you can see it in the Square dashboard
+                // at https://squareup.com/dashboard/customers/directory/all
+                // See more https://developer.squareup.com/reference/square/customers-api/create-customer
+                await customersApi.createCustomer(payload);
```
Comment (nicktrn) [inline src/square.ts:40]: Can't we just do this?

```suggestion
                await squareClient.customersApi.createCustomer(payload);
```

**Review by nicktrn — COMMENTED** [review]
Thanks for your submission! Left some comments. Also, if you could format this properly please. There are twice the amount of spaces (I think).

File: src/square.ts (hunk 1 above)
Comment (biplobsd) [reply src/square.ts:35]: Ok.. I did split because of makes more readable.

File: src/square.ts (hunk 2 above)
Comment (biplobsd) [reply src/square.ts:40]: Sure.

**Comment by biplobsd** [comment]
> Thanks for your submission! Left some comments. Also, if you could format this properly please. There are twice the amount of spaces (I think).

Thanks. Review again..

**Comment by nicktrn** [comment]
Looks good!

**Comment by D-K-P** [comment]
@biplobsd this is great, could you fix conflicts pls and ill merge

**Comment by biplobsd** [comment]
@D-K-P Fixed!

**Pull request merged**